### ISSUE TITLE
Missing Feature - "Control S" no longer works from v5.3.5-5.3.7 when saving changes.

### ISSUE THREAD

**Issue by AceTutor** [yuzoolcode/pulsecms-tracking#120]
One feature that has been with Pulse "forever" (in at least my windows environment) was the ability to use "Control S" to save changes in the Admin section. In my experience this changed at version 5.3.5, 5.3.6 and 5.3.7.

Can this be reviewed?

Hopefully I am not unique in having this experience and it is easily demonstrated elsewhere.

**Comment by ghost** [comment]
Not sure on this as not working on a Mac as that command saves the page but will look into this. Could be to do with the Redactor editor.

**Comment by pcalej** [comment]
Hey @AceTutor, having the "Ctrl+S" available is not only convenient but also makes a lot of sense. We assure you that we will look into what has changed in the newer versions so that we can restore it in our future updates.

**Comment by ghost** [comment]
Hey @AceTutor if it works it would in this demo. 
Can you confirm it's working? 
https://imperavi.com/redactor/examples/get-content/save-content-with-click-to-edit/

I'm worried it's been removed and have reported the change...

**Comment by AceTutor** [comment]
Hi Michael
I'm confused with your response.
From my perspective - when I visit the redactor site and test if their demonstration works my answer to you is "YES" it works! No doubt what I see is the same as you! That is a demo within their own site. Having similar features within Pulse is what we are about.

Since reading your response yesterday I have attempted to modify the "redactor_init.js" within Pulse Admin to see if I could mimic the redactor sites "SAVE" & "CANCEL" buttons and have the same working within my site. Regrettably I have not been successful. I am not a coder to be able to work out why not! 

My original plea to you was to investigate if the **"Ctrl+S"** feature can be restored. Is that something you or your Pulse Team can deal with or is this an Imperavi issue. I imagine it would not be too hard to establish via your license or liaison with Imperavi to find out if there is good reason for the change! Maybe there is a work around to achieve the same? Maybe its a feature that Imperavi has let slip under their radar during update? As I noted its a feature that  _has been with Pulse "forever"_.

Is this like my other "Missing Feature" - the **highlighting** switch noted in Issue 124?

From the Pulse Community perspective I hope these two issues don't get pushed into the too hard basket by yourself or Imperavi.

Again - thanks for a great product and support team approach you have implemented!

Cheers AceTutor

**Comment by ghost** [comment]
Ctrl+S is not working for me on:
https://imperavi.com/redactor/examples/get-content/save-content-with-click-to-edit/

Are you on Windows?

We've filed these as bug reports with them.
Also, can you replicate the highlighting? Again, not working for me on Safari on a Mac

**Comment by AceTutor** [comment]
I am on a windows system!

Crtl+S doesn't work on Imperavi site. I use both Chrome & Firefox browsers and Ctrl+S results in giving you the opportunity to save the page itself as an html file, etc.

What appears to be working is **SAVE** button. Upon clicking an alert says "The content saved!" I can only presume that if this was in a form the original would indeed be saved.

Can I recommend that you go back to an install of Pulse prior to version 5.3.5. Now edit a block, make a change and use Ctrl+S and it is saved. I have also just tested it with v4.6.2 and I am sure that it also works with all versions in between. As I said - a feature that has been with us "forever".

If you check version 5.3.4 you will also notice that if you highlight text then toggle the HTML button the highlighting does not remain! However from version 5.3.3 and earlier that is working!

Sorry - I don't have access to an Apple environment test this. I have just tried Safari on Windows (vs5.1.7) and notice that highlighting works but Ctrl-S does not nor does it save changes using the SAVE button using Pulse 5.3.3. (Apple no longer supports Safari on Windows so I am not surprised.)

To summarize - If you want both features in Windows environment you have to go back 5.3.3 or earlier to see these features in action! I trust that is also the same in your world!

Regards Ace

**Comment by ghost** [comment]
Works on PC and Mac with CTRL+S!
This is in 5.3.8
(As long as the edit box is selected with the Blue Box around it)

Like this:
https://imgur.com/KLkkauW

**Comment by ghost** [comment]
For the Text code highlighting I'm not sure what it is yet - have asked them to look at it.
The CTRL+S we added a hack for it to work.
